conda_package/mpas_tools/mesh/creation/sort_mesh.py

```diff
@@ -0,0 +1,187 @@
+
+import numpy as np
+import os
+import xarray
+import argparse
+from scipy.sparse import csr_matrix
+from scipy.sparse.csgraph import reverse_cuthill_mckee
+
+
+def sort_fwd(data, fwd):
```

Comment:
Let's indicate that this is "private":
```suggestion
def _sort_fwd(data, fwd):
```